__tests__/unstated.js

```diff
@@ -75,6 +76,50 @@ function CounterWithAmountApp() {
   );
 }
 
-test('basic', () => {
-  // still too lazy
+test('component with Subscribe renders fine', () => {
+  const wrapper = mount(
+    <Provider>
+      <Counter />
+    </Provider>
+  );
+  expect(wrapper).toMatchSnapshot();
+});
+
+test("component can change container's state", () => {
+  const wrapper = mount(
+    <Provider>
+      <Counter />
+    </Provider>
+  );
+  expect(wrapper).toMatchSnapshot();
+  const incButton = wrapper.find('button').last();
+  incButton.simulate('click');
+  incButton.simulate('click');
+  expect(wrapper).toMatchSnapshot();
```

Comment:
Any reason why you're using snapshots here instead of actually just making an explicit assertion? In my opinion, this is a common misuse of snapshots and I think it just makes it harder for developers to actually read what this test is actually trying to assert - therefore what is an acceptable snapshot diff?